projects/power_distribution/src/gpio.c

```diff
@@ -1,5 +1,5 @@
-#include "power_distribution_gpio.h"
-#include "power_distribution_events.h"
+#include "gpio.h"
```

Comment:
I don't think it's a good idea to rename `power_distribution_gpio` to just `gpio` since we already have a `gpio` library in ms-common, and the name conflicts. Perhaps change it to `pd_gpio` instead.